packages/cli/src/LoadNodesAndCredentials.ts

```diff
@@ -50,20 +40,20 @@ export class LoadNodesAndCredentials implements INodesAndCredentials {
 
 	includeNodes = config.getEnv('nodes.include');
 
-	credentialTypes: ICredentialTypes;
```

Comment:
this dependency tree was too messy. by moving code out into `CommunityPackagesService` and `FrontendService`, this class is a lot more leaner, and there are fewer cyclic dependencies now.
Also, the code that got moved out isn't loaded in `webhook` and `worker` services anymore.